### PR TITLE
Update Microsoft.WindowsAppSdk to 1.6 stable

### PR DESCRIPTION
**Pull request by Arlodotexe**
This PR updates our dependency on `Microsoft.WindowsAppSdk` to 1.6.240829007, the latest stable release. 

Prerequisite PR: https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/209

### CODE REVIEW

**Review by michael-hawker — APPROVED** [review]
Hopefully should go smoothly 🤞 approving conditional on tooling/build passing and submodule updated after it gets merged.

**Comment by Arlodotexe** [comment]
Seems okay from a build perspective and most components work fine in the Gallery, but a few components are crashing on load. 

Closing for now to keep us moving, but we need to do a full pass of the Gallery and resolve these issues. as part of https://github.com/CommunityToolkit/Tooling-Windows-Submodule/issues/205.

**Pull request merged**
